custom/up_nrhm/reports/asha_facilitators_report.py

```diff
@@ -3,11 +3,11 @@
 from corehq.apps.reports.standard import CustomProjectReport, DatespanMixin
 from corehq.apps.reports.util import format_datatables_data
 from custom.up_nrhm.sql_data import ASHAFacilitatorsData
-from django.utils.translation import ugettext_lazy as _, ugettext_noop
+from django.utils.translation import ugettext as _
 
 
 class ASHAFacilitatorsReport(GenericTabularReport, DatespanMixin, CustomProjectReport):
-    name = ugettext_noop("Format-2 Consolidation of the Functionality numbers")
+    name = _("Format-2 Consolidation of the Functionality numbers")
```

Comment:
Had the same discussion here: https://github.com/dimagi/commcare-hq/pull/7669/files#r35741689

Apparently it doesn't work